AccessibilitySnapshot/SnapshotTesting/Classes/SnapshotTesting+Accessibility.swift

```diff
@@ -0,0 +1,187 @@
+//
+//  Copyright 2020 Square Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SnapshotTesting
+
+extension Snapshotting where Value == UIView, Format == UIImage {
+
+    /// Snapshots the current view with colored overlays of each accessibility element it contains, as well as an
+    /// approximation of the description that VoiceOver will read for each element.
+    public static var accessibilityImage: Snapshotting {
+        return .accessibilityImage()
+    }
+
+    /// Snapshots the current view with colored overlays of each accessibility element it contains, as well as an
+    /// approximation of the description that VoiceOver will read for each element.
+    ///
+    /// - parameter showActivationPoints: When to show indicators for elements' accessibility activation points.
+    /// Defaults to showing activation points only when they are different than the default activation point for that
+    /// element.
+    /// - parameter useMonochromeSnapshot: Whether or not the snapshot of the `view` should be monochrome. Using a
+    /// monochrome snapshot makes it more clear where the highlighted elements are, but may make it difficult to
+    /// read certain views. Defaults to `true`.
+    /// - parameter markerColors: The array of colors which will be chosen from when creating the overlays
+    public static func accessibilityImage(
+        showActivationPoints activationPointDisplayMode: ActivationPointDisplayMode = .whenOverridden,
+        useMonochromeSnapshot: Bool = true,
+        markerColors: [UIColor] = []
+    ) -> Snapshotting {
+        guard isRunningInHostApplication else {
+            fatalError("Accessibility snapshot tests cannot be run in a test target without a host application")
```

Comment:
I think it's probably better to use a test failure here, rather than a fatal error. That way any other tests that are part of the same test suite (i.e. non-accessibility tests) will still get run. See `FBSnapshotTestCase+Accessibility.swift`:
```swift
guard isRunningInHostApplication else {
    XCTFail(
        "Accessibility snapshot tests cannot be run in a test target without a host application",
        file: file,
        line: line
    )
    return
}
```

Reply:
Turns out this isn't as straightforward as I hoped, since this method needs to return a `Snapshotting`. I thought maybe having it return an optional would work, but that complicates it all further. I think the cleanest thing for now is to keep the `fatalError`.